### ISSUE TITLE
Move main development back to `master`-branch

### ISSUE THREAD

**Issue by nknapp** [wycats/handlebars.js#1566]
Currently, we are developing on the  `4.x` branch, but everything has to be merged into master to keep it maintainable. This is not a lot of work, but a little bit frustrating. I have taken the time and searched through both logs. The main differences (i.e. commits that are part of master and not 4.x are):

## Breaking changes:

* Remove stringParams and trackIds mode 3f77b82ed00dc1d71976de76bd4f8dfb6e885f2a, 118836f390ffa882c720426db94e6b6748e69242
* Remove semi-documented _setup and _child APIs, 63a08890cce3620f3af328163f4313754df76581
* Remove amd files from the build, 8517352e209569f5a373d7a61ef4a673582d9616, 5302782c0842b26a151fd382b3bc9ba679c8a232

## Bugfixes
* Use objects for hash value tracking, 768ddbd6616f47a7a77ae2046775c43c2fe3e5ee

Everything else if chore or docs. @wycats keeping these commits on master would require us to bump to 5.0. I would probably remove support for lambdas (#1565) as well, when doing that.

**Comment by rwjblue** [comment]
FWIW, I believe that the divergence between `master` and `4.x` are partially to blame for the ergonomic regressions reported in https://github.com/wycats/handlebars.js/issues/1562 (specific explanation in https://github.com/wycats/handlebars.js/issues/1562#issuecomment-537295569).

**Comment by nknapp** [comment]
Might be, yes

**Comment by marado** [comment]
AFAICS, #1274 is also only merged on 5.0 but not on 4.x.

**Comment by nknapp** [comment]
Yes, that was before I told everyone to point PRs to the 4.x branch.

**Comment by ErisDS** [comment]
The master branch now also uses yargs instead of optimist/minimist, and has dropped support for Node.js < v10

**Comment by nknapp** [comment]
This is an implicit goal for 5.0, but the issue description documents parts of the changelog for 5.0, so I am assigning it to the milestone
